auth-web/src/components/auth/manage-business/EntityDetailsAlert.vue

```diff
@@ -0,0 +1,112 @@
+<!-- TODO:
+        get overlay colour
+        get alert icon colour
+        get header font and font size
+        compute array so that instead of mock arrays we get passed an array
+            of statuses then turn that array of statuses into an array of messages
+            then render those messages instead
+        align alert icon with bottom of status
+        change names of variables from status to something else
+         -->
+
+<template>
+   <v-tooltip top color="grey darken-4">
+     <template v-slot:activator="{ on }">
+       <v-icon :color=iconColour v-on="on">mdi-alert</v-icon>
+     </template>
+     <div class="py-2" :style="{'max-width': '300px'}">
+        <span>
+            {{ alertMessage }}
+        </span>
+        <ul>
+            <li v-for="message, i in detailMessages" :key="i">
+                {{ message.message }}
+            </li>
+        </ul>
+     </div>
+   </v-tooltip>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, nextTick, ref } from '@vue/composition-api'
+import { EntityDetailTypes } from '@/util/constants'
+import { PropType } from 'vue'
+
+export default defineComponent({
+  name: 'EntityDetailsAlert',
+  props: {
+    statuses: Array as PropType<Array<String>>
+  },
+  setup (props) {
+    // testing for now
+    const mockArray = ['FROZEN', 'BAD_STANDING', 'DISSOLUTION', 'LIQUIDATION'] as Array<String>
+
+    // actual stuff
+
+    const generateMessage = (status: String): { message: String, colour: String, priority: Number } => {
+      switch (status) {
+        case EntityDetailTypes.FROZEN:
+          return { message: 'This business is frozen', colour: '#F8661A', priority: 4 }
+        case EntityDetailTypes.BADSTANDING:
+          return { message: 'This business is not in good standing', colour: '#F8661A', priority: 3 }
+        case EntityDetailTypes.LIQUIDATION:
+          return { message: 'This business is in liquidation', colour: '#D3272C', priority: 2 }
+        case EntityDetailTypes.DISSOLUTION:
+          return { message: 'This business is in the process of being dissolved', colour: '#D3272C', priority: 1 }
+        default:
+          return null
+      }
+    }
+
+    const compareMessages = (m1, m2) => {
+      if (m1.priority < m2.priority) {
+        return -1
+      }
+      if (m1.priority > m2.priority) {
+        return 1
+      }
+      return 0
+    }
+
+    const detailMessages = computed(() => {
```

Comment:
I think you only need to run this once right? It will never change dynamically so I don't think you need to make this a computed value

Reply:
Yeah I don't think any user can mutate that, was just worried if the status of a business changed while a user was on the page. 

But the status would only change when a new call to LEAR or NAMEX is made on page reload so this only ran once